TreeDB/mongo_gateway/query_negation_test.go

```
@@ -0,0 +1,667 @@
+	choices := make(bson.A, mongoQueryMaxNegativeChoices)
+	for i := range choices {
+		choices[i] = int32(i)
+	}
+	nearCap, err := bson.Marshal(bson.D{{Key: "score", Value: bson.D{{Key: "$nin", Value: choices}}}})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if _, err := parseFindPredicates(wire.Document(nearCap)); err != nil {
+		t.Fatalf("near-cap $nin rejected: %v", err)
+	}
+	choices = append(choices, int32(mongoQueryMaxNegativeChoices))
```

golangci-lint (2.12.2)

[error] append to slice `choices` with non-zero initialized length

(makezero)